dist/js/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var _starter_app = require('./starter_app');
+
+var _starter_app2 = _interopRequireDefault(_starter_app);
```

Comment:
So the files in dist folder are created by Babel. Dont know what specifications it follows but our real code is in src folder.